packages/flutter/lib/src/material/ink_sparkle.dart

```diff
@@ -494,130 +495,31 @@ double _getTargetRadius(
 }
 
 // The code below is generated by the package: fragment_shader_manager.
-
+// It is hand modified to update for flutter/flutter style and asset loading.
```

Comment:
We should just update the `fragment_shader_manager` to generate the right code. AFAICT, it's only used for this purpose and nothing else?

Reply:
Yeah, we'll probably want to have a code gen solution, but it should likely be based on the reflection info emitted by impellerc and not on that package. So, spot fixes to `fragment_shader_manager` will probably not actually be too helpful.